lib/fortitude/rails/template_handler.rb

```diff
@@ -14,8 +14,10 @@ def call(template, &block)
 
         is_partial = !! File.basename(template.identifier) =~ /^_/
 
+        pathname = "#{template.identifier =~ %r(views/(.*)) && $1}"
```

Comment:
We need to pass the `@virtual_path` to the widget as per the comment above.

However, I'm not sure this code will work with views located outside of the normal view path of `$RAILS_ROOT/app/views`. What about views in other view paths, or in a gem?

This code was adopted from erector-rails4 (actually, I just noticed that I wrote it... woah...): https://github.com/ajb/erector-rails4/blob/d2fb36cd2f15d075b1f7c371b0c0978775b52908/lib/erector/rails/template_handler.rb#L6

...but it should hardly be considered bomb-proof.